src/inc/corprof.idl

```diff
@@ -125,6 +125,12 @@ import "wtypes.idl";
 import "unknwn.idl";
 #endif
 
+#ifdef PLATFORM_UNIX
```

Comment:
@jkotas Does this section need to be under `defined(_TARGET_X86_) && defined(PLATFORM_UNIX)`?